specification/deviceupdate/resource-manager/Microsoft.DeviceUpdate/preview/2020-03-01-preview/deviceupdate.json

```diff
@@ -1019,7 +1019,7 @@
           }
         },
         "identity": {
-          "$ref": "#/definitions/Identity",
+          "$ref": "../../../../../common-types/resource-management/v3/managedidentity.json#/definitions/ManagedServiceIdentity",
```

Comment:
Assuming you were just wanting to refactor this to use common types, you should get less breaking changes errors if you're refactoring to 'SystemAssignedServiceIdentity' instead of 'ManagedServiceIdentity' since your service didn't previously support user assigned identiy.

Reply:
Our RP is ready to support UserAssigned identity as well, that is why I wanted to include the reference including these. We are in public preview and no customers have adopted the identity feature yet.

Reply:
OK, my mistake, please just follow the breaking change guidance then.